spotinst_sdk2/clients/ocean/__init__.py

```diff
@@ -190,10 +206,262 @@ def get_aggregated_cluster_costs(self, ocean_id, aggregated_cluster_costs_reques
 
         aggregated_costs_response = self.send_post(
             body=body_json,
-            url=self.__base_ocean_url + "/" + ocean_id + "/aggregatedCosts",
+            url=self.__base_ocean_cluster_url + "/" + ocean_id + "/aggregatedCosts",
             entity_name='ocean (aggregated cluster costs)')
 
         formatted_response = self.convert_json(
             aggregated_costs_response, self.camel_to_underscore)
 
-        return formatted_response["response"]["items"][0]
\ No newline at end of file
+        return formatted_response["response"]["items"][0]
+
+    def initiate_roll(self, ocean_id: str, cluster_roll: aws_ocean.Roll):
+        """
+        Initiate Cluster Rolls
+
+        # Arguments
+        ocean_id (String): ID of the Ocean Cluster
+        cluster_roll (Roll): Roll with Instance Ids/ Launch specification Ids / None
+
+        # Returns
+        (Object): Cluster Roll API response
+        """
+        group_dict = aws_ocean.ClusterRollInitiateRequest(cluster_roll)
+
+        excluded_aggregated_costs_dict = self.exclude_missing(json.loads(group_dict.toJSON()))
```

Comment:
Variable names must be according to functionality or make it generic.

Reply:
Made generic variable